tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -31,15 +31,16 @@ func TestCreateDestroyWindowsInstance(t *testing.T) {
 	kubeconfig := os.Getenv("KUBECONFIG")
 	awscredentials := os.Getenv("AWS_SHARED_CREDENTIALS_FILE")
 	dir := os.Getenv("ARTIFACT_DIR")
+	privateKey := os.Getenv("SSH_PRIVATE_KEY")
```

Comment:
Is `SSH_PRIVATE_KEY` a path or the value itself? Because in the `CloudProviderFactory` function, the variable name is called `privateKeyPath`.

Reply:
It is a file.

Reply:
It's path. When you do a `os.GetEnv(somePath)`, you get value associated with it.

Reply:
Ya, you're right, I got the wrong environment variable.

Reply:
Done. PTAL